kinto_http/tests/test_session.py

```diff
@@ -18,6 +20,7 @@ def test_uses_specified_server_url(self):
 
     def test_no_auth_is_used_by_default(self):
         response = mock.MagicMock()
+        response.headers = {}
```

Comment:
nit: maybe a minimal `FakeResponse` class here would be better than repeating those `response.headers = ` assignations